kernel/arch/dreamcast/sound/snd_stream.c

```diff
@@ -106,6 +113,30 @@ static uint32 *sep_buffer[2] = {NULL, NULL};
 
 void snd_pcm16_split_sq_start(uint32_t *data, uintptr_t left, uintptr_t right, size_t size);
 
+static size_t samples_to_bytes(snd_stream_hnd_t hnd, size_t samples) {
```

Comment:
```suggestion
static inline size_t samples_to_bytes(snd_stream_hnd_t hnd, size_t samples) {
```
The compiler should really do this itself, but for small static functions like this, I just like to be damn sure... If you disagree, feel free to ignore this. :)